docs/ops/arithmetic/Acosh_3.md

```diff
@@ -6,32 +6,28 @@
 
 **Short description**: *Acosh* performs element-wise hyperbolic inverse cosine (arccosh) operation with given tensor.
 
-**Attributes**:
+**Detailed description**:  Operation takes one input tensor and performs the element-wise hyperbolic inverse cosine operation on a given input tensor, based on the following mathematical formula:
 
-    No attributes available.
+\f[
+a_{i} = acosh(a_{i})
+\f]
+
+**Attributes**: *Acosh* operation has no attributes.
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
```

Comment:
Is input tensor provided in radians or degrees?

Reply:
Which trigonometric operator provide such information?

Reply:
I saw it in tan op spec review (https://github.com/openvinotoolkit/openvino/pull/6567/files)

Reply:
Won't do if this is only in one trigonometric op. It should be one PR which update this in all.